css/main.css

```diff
@@ -325,3 +325,105 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* contact form styles */
```

Comment:
- [ ] You should remove all unused inline comments from your code. This helps make your codebase look more professional and clean, and also makes your code easily readable.